code/.clang-format

```diff
@@ -0,0 +1,6 @@
+BasedOnStyle: WebKit
+IndentWidth: 2
```

Comment:
I prefer 4 here, but I would rather have a any `.clang-format` file than none. So go ahead. Just if someone else thinks my way.

Do you think 2 is better for presenting exercises on projectors for group discussions? May actually be a good idea.